### PR TITLE
Update project management docs

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* Update documents to reflect the current status

### List of changes proposed in this PR (pull-request)

* Update ways of working
* Move some outdated docs in archive folder

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: ways_of_working.md [hunk 1]
```diff
@@ -7,65 +7,63 @@ It further outlines expectations and responsibilities with regard to working on
...
-- Record any new updates, exceptions or useful knowledge in a project management document
-- Document and share any conversation from closed spaces (such as email, Slack or 1:1 meeting) in a GitHub issue that could be useful for the community or community members in enabling their work in _The Turing Way_ 
+- documenting and sharing any conversation from closed spaces (such as email, Slack or 1:1 meeting) in a GitHub issue that could be useful for the community or community members in enabling their work in _The Turing Way_ 
 
 ## Communication
 
-The project members can be reached on [Slack](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw).
+You can reach out to all members by tagging them on GitHub issues or Pull Requests or on [Slack](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw).
 
 You can reach the PI and Co-I of the project through their preferred way of communication:
```
Comment (KirstieJane) [inline ways_of_working.md:45]:
```suggestion
You can reach the lead investigators of the project through their preferred way of communication:
```

File: ways_of_working.md [hunk 2]
```diff
@@ -7,65 +7,63 @@ It further outlines expectations and responsibilities with regard to working on
...
 | Esther Plomp | Faculty of Applied Sciences at Delft University of Technology - Data Steward | in-kind contribution | Feb 2020 - Present | - |
-| Sarah Gibson | 2i2c Research Software Engineer/JupyterHub Community Development | in-kind contribution | - | Prevously: Research Data Scientist, Nov 2018 - Aug 2021 (40%) |
-| May Yong | Research Associate | 40% | Oct 2021 - Dec 2021 | - |
-| Lydia France | Research Associate | 50% | Oct 2021 - Dec 2021 | - |
-| Iain Stanson | Research Associate | 30% | Nov 2021 - Dec 2021 | - |
+| Sarah Gibson | 2i2c Open Source Infrastructure Engineer/JupyterHub Community Development | in-kind contribution | - | Prevously: Research Data Scientist, Nov 2018 - Aug 2021 (40%) |
+| May Yong | Research Software Engineer | 40% | Oct 2021 - Dec 2021 | - |
+| Lydia France | Research Software Engineer | 50% | Oct 2021 - Dec 2021 | - |
+| Iain Stanson | Research Software Engineer | 30% | Nov 2021 - Dec 2021 | - |
 | Paul Owoicho | Google Season of Docs - Technical Writer | 100% | Sep 2020 - Dec 2020 | - |
```
Comment (KirstieJane) [inline ways_of_working.md:20]: What do you think about formatting this table to be clearer about separating previous folks?

Not a strong opinion, if you think the dates are enough then that's totally fine with me.

**Review by KirstieJane — APPROVED** [review]
Looks great to me! A couple of comments but unimportant ones.

File: ways_of_working.md (hunk 2 above)
Comment (malvikasharan) [reply ways_of_working.md:20]: For this document, it makes sense to put them in separate tables so people can identify who to contact.

**Comment by malvikasharan** [comment]
I will merge it and share with the CMs on Slack. Thank you for the quick review @KirstieJane 🙌

**Pull request merged**